config/crd/bases/configuration.konghq.com_kongingresses.yaml

```diff
@@ -237,6 +237,12 @@ spec:
                       concurrency:
                         minimum: 1
                         type: integer
+                      headers:
+                        additionalProperties:
+                          items:
+                            type: string
+                          type: array
+                        type: object
```

Comment:
Where are those changes coming from 🤔 ?

Reply:
It has changes in the manifests due to a cascading update of go-kong which introduced a new field in v0.43.0: https://github.com/Kong/go-kong/pull/331.